tests/testthat/test-combine-categories.R

```diff
@@ -123,6 +123,27 @@ with_mock_crunch({
             list(list(name="Both", categories=NULL))),
             "Combinations must reference 'categories' by name or id")
     })
+
+    test_that("collapseCategories errors correctly", {
+        expect_error(collapseCategories(ds$birthyr, "from", "to"),
+            "Variable must be a categorical.")
+        expect_error(collapseCategories(ds$gender, "Male", 1),
+            "Destination category must be a character string of length 1.")
+        expect_error(collapseCategories(ds$gender, "Male", c("young", "old")),
+            "Destination category must be a character string of length 1.")
+        expect_error(collapseCategories(ds$gender, 1, "young"),
+            paste0(sQuote('from'), " must be a character vector or logical expression."))
+    })
+    test_that("collapseCategories produces the expected patch", {
+        expect_PATCH(var <- collapseCategories(ds$location, c("London", "Scotland"), "GB"),
+            "https://app.crunch.io/api/datasets/1/variables/location/",
+            '{"categories":[{"id":1,"missing":false,"name":"London","numeric_value":1},{"id":2,"missing":false,"name":"Scotland","numeric_value":2},{"id":-1,"missing":true,"name":"No Data","numeric_value":null},{"id":3,"name":"GB","numeric_value":3}]}'
+        )
+        expect_POST(var <- collapseCategories(ds$location, c("London", "Scotland"), "London"),
```

Comment:
Be clear that the "expected PATCH" here is that you aren't including "London" in the update command (though what would the harm be of including it?)